### PR TITLE
[plugin-web-app] Use instant scroll behavior for screenshots taking

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: vividus-plugin-web-app/src/main/java/ru/yandex/qatools/ashot/shooting/DebuggingViewportPastingDecorator.java [hunk 1]
```diff
@@ -39,6 +39,8 @@
  */
 // CHECKSTYLE:OFF
 public class DebuggingViewportPastingDecorator extends ShootingDecorator {
+    private static final String SCROLL_Y = 
+            ".scrollTo({\"top\":arguments[0], \"left\": 0, \"behavior\":\"instant\"}); return [];";
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/ru/yandex/qatools/ashot/shooting/DebuggingViewportPastingDecorator.java:43]: IE doesn't support this API and only the latest version (24.03.2020) of Safari supports it: https://caniuse.com/#search=scrollTo

Comment (ikalinin1) [reply vividus-plugin-web-app/src/main/java/ru/yandex/qatools/ashot/shooting/DebuggingViewportPastingDecorator.java:43]: Do you agree to use it for the browsers supporting the API?

Comment (valfirst) [reply vividus-plugin-web-app/src/main/java/ru/yandex/qatools/ashot/shooting/DebuggingViewportPastingDecorator.java:43]: yes, sounds reasonable

File: vividus-plugin-web-app/src/main/java/ru/yandex/qatools/ashot/shooting/DebuggingViewportPastingDecorator.java [hunk 2]
```diff
@@ -39,11 +39,27 @@
...
+            + "return [];";
     private static final long serialVersionUID = 4173686031614281540L;
     private static final String CURRENT_SCROLL = "current_scroll_";
     protected int scrollTimeout = 0;
     private Coords shootingArea;
     private transient ScreenshotDebugger screenshotDebugger;
+    protected final String scrollYScript;
+
+    {
+        scrollYScript = String.format(SCROLL_Y_TEMPLATE, getTarget());
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/ru/yandex/qatools/ashot/shooting/DebuggingViewportPastingDecorator.java:61]: what about moving this part to constructor and converting `getTarget()` to a constructor parameter?

**Pull request merged**